### PR TITLE
Image: Check cache availability before rendering images as initially hidden

### PR DESCRIPTION
**Pull request by asmundg**
#### Pull request checklist

- [x] Addresses an existing issue: #1041
- [x] Include a change request file if publishing 
- [x] New enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

Before assuming that there is a delay before an image can be shown, check if it exists in local cache. If it does, make the image visible immediately, instead of hiding it until the onLoad event.

This removes the textboy flicker when mounting Personas with cached profile images.

#### Focus areas to test

Persona, Image. See http://codepen.io/asmundg/pen/YNoaRG.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Image/Image.tsx [hunk 1]
```diff
@@ -124,11 +127,18 @@ export class Image extends BaseComponent<IImageProps, IImageState> {
...
     if (src) {
       this.setState({
-        loadState: ImageLoadState.loaded
+        loadState: ImageLoadState.loaded,
+        cached: true
       });
     }
   }
 
+  private _isImageCached(src: string): boolean {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Image/Image.tsx]: This looks like a performance concern. Now every time the component receives props, it's creating an img element?

Comment (asmundg) [inline packages/office-ui-fabric-react/src/components/Image/Image.tsx]: _If_ src changes. But yes, this is a potential issue.

Comment (dagoff) [inline packages/office-ui-fabric-react/src/components/Image/Image.tsx]: I agree with dzearing.

Comment (asmundg) [inline packages/office-ui-fabric-react/src/components/Image/Image.tsx]: Alternative take, with configuration instead of computation: #1075

**Comment by asmundg** [comment]
Dropping this in favour of #1075.

**Pull request closed without merging**